### ISSUE TITLE
SchemaBuilder.Setup.CompleteSchema outputs unnecessary text

### ISSUE THREAD

**Issue by msawczyn** [ChilliCream/hotchocolate#4573]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

v12.4.1

In `SchemaBuilder.Setup.CompleteSchema`, there is a 
```
Console.WriteLine("Types: " + typeRegistry.Types.Count);
```
that interferes with a common use case of calling `ISchema.Print` and sending the result to `stdout`. Since this is unnecessary, I'd suggest changing it to either a `Debug.WriteLine` call or removing it completely. The extra text pollutes any output the programmer intends to capture (e.g., to a file to be consumed by other tools).

### Steps to reproduce

```
      var host = hostBuilder.Build();

      var executor = host.Services
         .GetRequiredService<IRequestExecutorResolver>()
         .GetRequestExecutorAsync()
         .Result;

      var schema = executor.Schema.Print();
      Console.WriteLine(schema);
```

### Product

Hot Chocolate

### Version

12.4.1

**Comment by msawczyn** [comment]
Please ignore.

**Comment by tobias-tengler** [comment]
Michael removed this two days ago: https://github.com/ChilliCream/hotchocolate/commit/20ed41f7fb6d0c8df80ad44ee14e65ea010cdf35

And according to the changelog it was released with v12.4.1: https://github.com/ChilliCream/hotchocolate/releases/tag/12.4.1

Are you sure you are on v12.4.1?

**Comment by msawczyn** [comment]
Heh. Nope. Not sure at all 😁 . But I am now.

Closing....
